### ISSUE TITLE
Remote video error

### ISSUE THREAD

**Issue by IvanDhalluin** [mifi/reactive-video#5]
Hello,
I'm getting `TypeError: must pass in a file:// URI to convert to a file path` when trying to use the code from your exemple :
`<Video src="https://static.mifi.no/Zv5RvLhCz4M-small.mp4" />`

I tried with or without headless and with different version of @reactive-video/builder

Do you have any clue?

**Comment by mifi** [comment]
can you confirm you're using the newest version of both packages? also where is the error coming from?

**Comment by IvanDhalluin** [comment]
I'm using @reactive-video/builder 1.7.0 with node 14.17 and ffmpeg 4.04.
I'm using exemples codes from your Readme. Here are the logs :
```
Compiling Reactive Video Javascript
Starting server
Launching puppeteer
Rendering with concurrency 1
TypeError: must pass in a file:// URI to convert to a file path
    at fileUriToPath (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/file-uri-to-path/dist/src/index.js:14:15)
    at /home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/@reactive-video/builder/server.js:58:18
    at asyncUtilWrap (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express-async-handler/index.js:3:20)
    at Layer.handle [as handle_request] (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express/lib/router/layer.js:95:5)
    at /home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express/lib/router/index.js:341:12)
    at next (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/express/lib/router/index.js:275:10)
page 0 log Failed to load resource: the server responded with a status of 500 (Internal Server Error)
Caught error at frame 0, part 0 (0) Error: Render frame error: HTTP error
    at renderFrame (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/@reactive-video/builder/index.js:229:63)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async /home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/p-timeout/index.js:50:13
Output ffmpeg exited with code 1
Stopping bundle watcher
Bundle watcher stopped
Error: Render frame error: HTTP error
    at renderFrame (/home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/@reactive-video/builder/index.js:229:63)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async /home/ivandhalluin/Documents/test-reactive-video/awesome-reactive-video/node_modules/p-timeout/index.js:50:1
```

**Comment by mifi** [comment]
Thanks! There was indeed a bug there. I hadn't tested with HTTP URLs (I mostly use local files, `file://` for my projects)
Should be fixed in the newest version 1.7.1

**Comment by IvanDhalluin** [comment]
Thanks!
It's working now :muscle:
